### PR TITLE
consolidating vis pages into one page.

### PR DESCRIPTION
**Pull request by adphillips2017**
needs testing.  making draft pr for feedback.

### CODE REVIEW

File: website/src/app/pages/visualization/visualization.component.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+@Component({
+  selector: 'agc-visualization',
+  templateUrl: './visualization.component.html',
+  styleUrls: ['./visualization.component.scss']
+})
+export class VisualizationComponent implements OnInit {
+  visualizationID = '';
+  private routeSub: Subscription = {} as Subscription;
+  visualizations: VisualizationDictionary = {
```
Comment (bherr2) [inline website/src/app/pages/visualization/visualization.component.ts:29]: Move these visualizations into ngxs state and actual config data into its own .ts file (ideally its json, but we need support for creating functions that you can't do in json as it is).

Comment (bherr2) [reply website/src/app/pages/visualization/visualization.component.ts:29]: Look up routing to see how you can prefetch this data (using a route guard)

Comment (bherr2) [reply website/src/app/pages/visualization/visualization.component.ts:29]: Look up routing to see how you can prefetch this data (using a route guard)

**Review by axdanbol — APPROVED** [review]
Looks good

**Pull request merged**